mava/__init__.py

```diff
@@ -13,7 +13,7 @@
 from mava.core import Executor, Trainer
 
 # Expose the environment loop.
-from mava.environment_loop import ParallelEnvironmentLoop, SequentialEnvironmentLoop
+from mava.environment_loop import ParallelEnvironmentLoop
```

Comment:
Are we still going to support Sequential env loops in the future? If not, or at least if not in the near to medium term future, I guess we can simply call this class `EnvironmentLoop` and drop the "Parallel"?

Reply:
I will check with @DriesSmit, if we won't support it I will change the name!

Reply:
Given our discussion offline, let's keep it as is for now 👍